CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+### Fixed
+- Changed the import path for `go.uber.org/atomic` to its newer, canonical
+  import path.
+  [#18](https://github.com/uber-go/ratelimit/issues/18)
+
+## [1.0.0] - 2016-09-09
```

Comment:
don't think we ever released anything?

Reply:
I’m tagging what stands on master so we can track.

Reply:
backed this out. will follow with 0.1 tag.